### ISSUE TITLE
What's the setting of success rate reported in RoboCasa with Franka

### ISSUE THREAD

**Issue by ZhaoRunyi** [NVIDIA/Isaac-GR00T#506]
### 📚 The doc issue

Hi, thanks to your great work!

I wonder what's the setting of RoboCasa with  Franka success rate you reported in https://github.com/NVIDIA/Isaac-GR00T/blob/main/examples/robocasa/README.md

1. Is it fine-tuned or zero-shot?
2. If it's finetuned, is it multi-task or one-checkpoint-for-one-task?
3. Details in finetune process, like data volumn (and ids to the trajectories used), hyper-paramters (learning rate schedule, batch size, epochs, random seeds), for us to reproduce the result.

I would deeply appreciate these details provided, and I believe it will definitely contribute to the reproductivity of your model.
